common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -41,6 +41,15 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  val PlistaForOutbrainAU = Switch(
```

Comment:
If this switch is for the replacement of outbrain for plista, it should have an expiry date.
Then we will need another permanent switch to turn plista on or off when it goes wrong.